eng/Publishing.props

```diff
@@ -10,14 +10,25 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <PublishDependsOnTargets>$(PublishDependsOnTargets);_PublishInstallers</PublishDependsOnTargets>
+    <PublishDependsOnTargets>$(PublishDependsOnTargets);_PublishInstallers;_PublishDashboardAssets</PublishDependsOnTargets>
   </PropertyGroup>
 
   <ItemGroup>
     <_InstallersToPublish Include="$(ArtifactsDir)**\*.wixpack.zip" Condition="'$(PostBuildSign)' == 'true'" />
     <_InstallerManifestFilesToPublish Include="$(ArtifactsDir)VSSetup\$(Configuration)\Insertion\**\*.zip" />
+    <_DashboardFilesToPublish Include="$(DashboardPublishedArtifactsOutputPath)\**\*.zip" />
```

Comment:
nit: this can be moved into the new target, and we can check that it is not empty.

Reply:
Actually I had to remove this due to the fact that we currently (incorrectly) call publish more than once, and the first time we do so we haven't yet created the publish assets, so this is failing. I'll log an issue to track the work of simplifying our build (today it requires one call to build.cmd and one manual build of the workloads.proj, but ideally we would only need to call build.cmd). Once we move to that model, then we can add the check back.